index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
+    <link href="https://fonts.googleapis.com/css2?family=Cairo:wght@200&family=Gudea:wght@400;700&family=IBM+Plex+Sans:wght@300;400;500&family=Lato:wght@300;400;700&family=Open+Sans:wght@300;400;500&family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSK2rfCGbrDopSuC8R5mnnRymSX0r1mVkqQAEiwNh6mHMDNfQ4NTHlySOnrq7B_XZEjhlU&usqp=CAU" id="header-img">
+        <nav id="nav-bar">
+      
+            <ul>
+              <li><a href="#features" class="nav-link">Features</a></li>
+              <li><a href="#video" class="nav-link">video</a></li>
+              <li><a href="#prices" class="nav-link">Prices</a></li>
+              <li><a href="#form" class="nav-link">form</a></li>
+            </ul>
+        </nav>
+    </header>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+        <h2>Logitech G915 The best on the market</h2>
+        <input type="email" name="email" id="email" placeholder="Please enter your email" required>
+        <input type="submit" value="Submit" id="submit">
+    </form>
+    <section id="feature__section">
+        <div id="features">
```

Comment:
- [ ] I think it would be better to remove this `div` as it is the only child of `feature__section` section. then add styles to children by selecting `feature__section` then targeting its children.